### PR TITLE
Need `LinearAlgebra` to use `I`

### PR DESCRIPTION
**Pull request by likanzhan**
Need `LinearAlgebra` to use `I`

### CODE REVIEW

**Comment by StefanKarpinski** [comment]
Can you add blank lines after each of these since that's how the REPL prints them? (Note that this breaks the doctests, so you may want to make sure it doesn't do that.)

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 1]
```diff
@@ -98,6 +98,9 @@ Returns the number of stored (filled) elements in a sparse array.
 
 # Examples
 ```jldoctest
+
```
Comment (StefanKarpinski) [inline stdlib/SparseArrays/src/sparsematrix.jl:101]:
```suggestion
```

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 2]
```diff
@@ -130,6 +133,9 @@ modifications to the returned vector will mutate `A` as well. See
...
```
Comment (StefanKarpinski) [inline stdlib/SparseArrays/src/sparsematrix.jl:136]:
```suggestion
```

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 3]
```diff
@@ -158,6 +164,9 @@ nonzero values. See also [`nonzeros`](@ref) and [`nzrange`](@ref).
...
```
Comment (StefanKarpinski) [inline stdlib/SparseArrays/src/sparsematrix.jl:167]:
```suggestion
```

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 4]
```diff
@@ -664,6 +673,9 @@ Convert an AbstractMatrix `A` into a sparse matrix.
...
```
Comment (StefanKarpinski) [inline stdlib/SparseArrays/src/sparsematrix.jl:676]:
```suggestion
```

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 5]
```diff
@@ -3322,6 +3334,9 @@ Concatenate matrices block-diagonally. Currently only implemented for sparse mat
...
```
Comment (StefanKarpinski) [inline stdlib/SparseArrays/src/sparsematrix.jl:3337]:
```suggestion
```

**Review by StefanKarpinski — CHANGES_REQUESTED** [review]
No blank lines above the first prompt. In general: take a look at how the surrounding code and/or docs look and emulate it.

**Comment by Keno** [comment]
Would it be better to just have SparseArrays re-export `I`?

**Comment by StefanKarpinski** [comment]
Let's see if tests pass these days.

**Comment by KristofferC** [comment]
You need to rebase it since the buildbots don't run on the merge commit.

**Comment by vtjnash** [comment]
We already do this, via `make.jl`, so if we think it should be in each example, we should remove it from the global config:
```
DocMeta.setdocmeta!(
    SparseArrays,
    :DocTestSetup,
    maybe_revise(:(using SparseArrays, LinearAlgebra));
    recursive=true, warn=false,
)
```

I'm inclined to instead reject this PR, but I'm not opposed to it if others feel it is reads better.

**Pull request closed without merging**